src/navigation/actions.js

```diff
@@ -402,17 +401,8 @@ export const createSupplierRequisition = ({
   currentUser,
   ...requisitionParameters
 }) => dispatch => {
-  // Fetch this stores custom data to find if this store has customized
-  // monthsLeadTime.
-  const customData = Settings.get(SETTINGS_KEYS.THIS_STORE_CUSTOM_DATA);
-
-  // CustomData is a stringified JSON object.
-  const parsedCustomData = customData ? JSON.parse(customData) : '';
-
   // Months lead time has an effect on daysToSupply for a requisition.
-  const monthsLeadTime = parsedCustomData.monthsLeadTime
-    ? Number(parsedCustomData.monthsLeadTime.data)
-    : 0;
+  const monthsLeadTime = UIDatabase.getPreference(PREF_KEYS.MONTHS_LEAD_TIME);
```

Comment:
Exact same behaviour but to be more explicit about the value passed to `createRecord`, could `monthsLeadTime = ...getPref... ?? 0`

Reply:
9082642ad73a41a14c55a0096c15c0c53573560e. Also added explicit default assignment behaviour with pref type refactor (see `parseNumericData` etc).